### PR TITLE
RF 034.001 MainPageSpec | <Header > verify "For Business" button

### PR DESCRIPTION
**Pull request by AlexMokeev**
RF |  MainPageSpec> AT_034.001 | <Header > verify "For Business" button

### CODE REVIEW

**Comment by AlexMokeev** [comment]
https://trello.com/c/eqO9KkxY/757-rf-mainpagespec-at034001-header-verify-for-business-button

File: cypress/e2e/mainPageSpec.cy.js [hunk 1]
```diff
@@ -40,4 +40,12 @@ describe('mainPageSpec', () => {
     it('AT_045.006 | Main page > Section with 8-day forecast > Verifying the weather forecast for 8 days is displayed in the section', function () {
         mainPage.elements.getForecastDays().should('have.length', this.data.forecastDaysLength);
     });
+    
+    it('AT_034.001 | <Header > verify "For Business" button', function () {
+        mainPage.elements
```
Comment (Rom-T) [inline cypress/e2e/mainPageSpec.cy.js:45]: without elements. => mainPage.clickForBusinessBtn()

File: cypress/e2e/mainPageSpec.cy.js [hunk 2]
```diff
@@ -40,4 +40,12 @@ describe('mainPageSpec', () => {
...
+        .clickForBusinessBtn
+
+        mainPage.elements
```
Comment (Rom-T) [inline cypress/e2e/mainPageSpec.cy.js:48]: The same

File: cypress/pageObjects/MainPage.js [hunk 3]
```diff
@@ -27,5 +28,14 @@ class MainPage {
     clickHomePageButton() {
         this.elements.getHomePageButton().click({force: true});
     }
+
+    clickForBusinessBtn() {
+        this.elements.getForBusinessButton()
+        .invoke(removeAttr, target)
+        .click()
```
Comment (Rom-T) [inline cypress/pageObjects/MainPage.js:35]: add to click {force: true}

File: cypress/pageObjects/MainPage.js [hunk 4]
```diff
@@ -27,5 +28,14 @@ class MainPage {
...
+    }
+    checkUrlForBusiness() {
+        cy.url().should('eq', 'https://openweather.co.uk/')
```
Comment (Rom-T) [inline cypress/pageObjects/MainPage.js:38]: should be in the test, not in the page class

File: cypress/pageObjects/MainPage.js (hunk 3 above)
Comment (AlexMokeev) [reply cypress/pageObjects/MainPage.js:35]: fixed it

**Review by Rom-T — DISMISSED** [review]

File: cypress/pageObjects/MainPage.js (hunk 4 above)
Comment (AlexMokeev) [reply cypress/pageObjects/MainPage.js]: fixed it

**Comment by AlexMokeev** [comment]
падают многократно тесты...

**Pull request closed without merging**